packages/react-native-editor/ios/GutenbergDemo/GutenbergViewController.swift

```diff
@@ -27,7 +27,6 @@ class GutenbergViewController: UIViewController {
         super.viewDidLoad()
         configureNavigationBar()
         gutenberg.delegate = self
-        navigationController?.navigationBar.isTranslucent = false
```

Comment:
This specific color is not really necessary and the approach is incompatible with iOS 15.